### DISCUSSION TITLE
How do I remove all global cache?

### DISCUSSION THREAD

**Discussion by xq114** [xmake-io/xmake#2277] [Q&A]
如何清除全局缓存：~/.xmake/cache/detect等文件？xmake f -c 和xmake clean --all 都只对当前项目生效

**Comment by waruqi** [comment]
xmake g -c

**Reply by xq114** [reply]
但一些global的设置（proxy_pac等）并不想清理，只想清理缓存

**Reply by waruqi** [reply]
目前没办法，要么你删下文件 这种暂时就不单独开args了
